README.md

```diff
@@ -1,264 +1,505 @@
+# NAMETBD: A design system integrating to Pattern Lab and a Drupal 8 theme
```

Comment:
Interesting philosophical point there. Does the design system integrate with these, or does the design system generate these?

Reply:
These apps exist without the DS. They just look like crap/fail to compile due to missing file references.